.github/workflows/deploy-to-eks.yaml

```
@@ -12,6 +12,9 @@ on:
       uats_branch:
         description: 'Branch to run the UATs from e.g. main or track/1.9. By default, this is defined by the dependencies.yaml file.'
         required: false
+      bundle_url:
+        description: 'URL to be used for version checking. In tracks prior to 1.10 release (track/1.8 and track/1.9), its value is hardcoded in the UATs. For main and track/1.10 branches this value can customized, despite sensible defaults are provided. Leave this empty unless you want to perform a custom check.'
```

```suggestion
        description: 'URL to be used for version checking. In tracks prior to 1.10 release (track/1.8 and track/1.9), its value is hardcoded in the UATs. For main and track/1.10 branches this value can be customized, but sensible defaults are provided. Leave this empty unless you want to perform a custom check.'
```
nit: I think `despite` needs a noun afterwards, hence the swap with `but`. Still, this is a suggestion, feel free to phrase it as you prefer.